ui/apps/ui/src/app/components/articles/articles.component.ts

```diff
@@ -16,27 +16,10 @@ export class ArticlesComponent {
   articles: IArticle[] = [
     {
       id: '0',
-      title: 'Exploration Toolkit',
-      description: 'Navigating EOSC<br>Catalogue & Marketplace',
-      url: `${
-        this._configService.get().knowledge_hub_url
-      }/articles/article_01.html`,
-    },
-    {
-      id: '1',
-      title: 'Searching Pathways',
-      description: 'Smart Strategies<br>for EOSC Exploration',
-      url: `${
-        this._configService.get().knowledge_hub_url
-      }/articles/article_02.html`,
-    },
-    {
-      id: '2',
-      title: 'Effective Data Reuse',
-      description: 'Documentation and Metadata Refinement',
-      url: `${
-        this._configService.get().knowledge_hub_url
-      }/articles/article_03.html`,
+      title: 'EOSC Knowledge Hub',
+      description:
+        'Discover training materials, courses and learning paths supporting open science and FAIR sharing ',
+      url: `https://beta.knowledge-hub.eosc-portal.eu/ `,
```

Comment:
Please replace that with a variable `KNOWLEDGE_HUB_URL`: https://github.com/cyfronet-fid/eosc-search-service/blob/eb3bbd0b19bcc978eb949794a75cd251b49c701b/README.md?plain=1#L185